Azure-Samples/communication-services-javascript-quickstarts
# [ui-library-add-call-recording] Convert backend to .NET Azure Functions

## Purpose

Call automation SDKs are only available in Java and C#.

## How to Test

See `./ui-library-add-call-recording/README.md` for instructions for running locally.
